web/public/js/directives/cd-rounded-icon/style.less

```diff
@@ -0,0 +1,16 @@
+.cd-rounded-icon{
+  .icon {
```

Comment:
In keeping with [BEM](http://getbem.com/introduction/), this should be &__icon to scope your styles only to this directive, and not any of it's children. (I know this directive won't have children, but consistency.